nuxeo-services/nuxeo-platform-convert/src/test/java/org/nuxeo/ecm/platform/convert/tests/TestZip2HtmlConverter.java

```diff
@@ -60,14 +60,20 @@ public void testConvertToMimeType(String mimeType) throws IOException {
         assertNotNull(blobs);
         assertEquals(3, blobs.size());
 
+        Map<String, String> zipContent = new HashMap<>();
+        zipContent.put("hello.xml", "Hello from a xml <b>document</b>");
+        zipContent.put("hello.txt", "Hello from a text document");
+
         String content = blobs.get(0).getString();
         assertTrue(content, content.contains("<li><a href=\"hello.xml\">hello.xml</a></li>"));
         assertTrue(content, content.contains("<li><a href=\"hello.txt\">hello.txt</a></li>"));
 
-        content = DocumentUTUtils.readContent(blobs.get(1).getFile());
-        assertTrue(content, content.contains("Hello from a xml <b>document</b>"));
-
-        content = DocumentUTUtils.readContent(blobs.get(2).getFile());
-        assertTrue(content, content.contains("Hello from a text document"));
+        for (int i = 1; i < blobs.size(); i++) {
+            File file = blobs.get(i).getFile();
+            content = DocumentUTUtils.readContent(file);
+            assertTrue(content, content.contains(zipContent.get(file.getName())));
+            zipContent.remove(file.getName());
+        }
+        assertEquals(zipContent.size(), 0, zipContent.size());
```

Comment:
Could you replace it by an `assertTrue` with a call on `isEmpty` ? Message could be something about expected files not found.

Reply:
Yes like `assertTrue("Unexpected remaining files: " + zipContent.keys(), zipContent.isEmpty())`

Reply:
Yes, much clear 👍 thanks